.travis.yml

```diff
@@ -1,6 +1,17 @@
 sudo: required
 script: make travis
+# To use docker buildx for multi architexture builds, the latest version of
+# docker must be installed and the experimental features enabled.
+# Refer to: https://medium.com/@quentin.mcgaw/cross-architecture-docker-builds-with-travis-ci-arm-s390x-etc-8f754e20aaef
+before_install:
+  - curl -fsSL https://get.docker.com | sh
+  - echo '{"experimental":"enabled"}' | sudo tee /etc/docker/daemon.json
+  - mkdir -p $HOME/.docker
+  - echo '{"experimental":"enabled"}' | sudo tee $HOME/.docker/config.json
+  - sudo service docker start
```

Comment:
This seems less than ideal to have to do on each run. From the docker buildx docs..

> These features are intended for testing and feedback only as they may change between releases without warning or can be removed entirely from a future release. Experimental features must not be used in production environments. Docker does not offer support for experimental features.

And well, IMO, the OPA build is a production use-case.

Any idea when this feature would be GA?

As long as we're looking at experimental features, any reason we can't just add and arm target via https://docs.travis-ci.com/user/multi-cpu-architectures/ and do the whole thing with unit testing and everything?

Curious what other folks think.. /cc @tsandall @ashutosh-narkar

Reply:
Hi, the travis documentation states that it is alpha at the moment.

Architectures currently supported:
 amd64
 ppc64le
 s390x
 arm64

There is no arm/v6 (Raspberry Pi) architecture option for this, while the bulldx option does support this.

Will see if I can find anything on buildx going GA